app/src/main/java/org/mozilla/fenix/compose/cfr/CFRPopupFullscreenLayout.kt

```diff
@@ -70,7 +70,7 @@ private data class PopupHorizontalBounds(
  * @param action Optional other composable to show just below the popup text.
  */
 @SuppressLint("ViewConstructor") // Intended to be used only in code, don't need a View constructor
-internal class CFRPopupFullScreenLayout(
+internal class CFRPopupFullscreenLayout(
```

Comment:
`Fullscreen` is the correct naming. I prefer renaming a class to renaming a file, given the choice, as it also preserves the history better.

Reply:
`Fullscreen` is fine. However, I am confused about the latter statements - renaming a file shouldn't affect history. This is illustrated in https://github.com/mozilla-mobile/fenix/pull/26851/commits/ffae5e89936e5bae3f6ded790ff58451280b818f. The trick is to ensure you have a commit that simply renames the file without additional changes, otherwise, git will consider it a brand new file.